### ISSUE TITLE
Update Credits before a new release

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#54]
Here are the current credits on the website. I'm not seeing anything about QA. I bet that needs updating.

![image](https://user-images.githubusercontent.com/6856943/27511427-365940c8-58d0-11e7-9656-e75a3366b9b8.png)

**Comment by zepumph** [comment]
from code review: https://github.com/phetsims/ohms-law/issues/51

**Comment by zepumph** [comment]
@ariel-phet do these credits seem good to you?

**Comment by ariel-phet** [comment]
@zepumph the general credits seem right. Kathy should be added to the "Team" and as you mentioned QA should be added in the new release. Did Martin do some refactoring that should be credited?

**Comment by zepumph** [comment]
I should add `Alex Dornan, Ethan Johnson`  to the qa credits. I will wait to see if more needs to change.

**Comment by jessegreenberg** [comment]
Lets update this working comment for additions to credits (not yet alphabetized):

QA: Alex Dornan, Ethan Johnson, Kathryn Woessner
Software Development: Michael Kauzmann, Martin Veillette
Team: Kathy Perkins, Emily Moore, Taliesin Smith

**Comment by jbphet** [comment]
I was asked to comment on the credits in https://github.com/phetsims/QA/issues/313#issuecomment-488394000.  The proposal seems fine to me.  Please keep M Dubson first amongst the SW developers, since he authored the original sim and I'd like to acknowledge that.

**Comment by jessegreenberg** [comment]
Thanks @jbphet, that sounds great.

**Comment by KatieWoe** [comment]
Ethan Ward should be added as well.

**Comment by jessegreenberg** [comment]
Credits updated in master and 1.4

**Comment by KatieWoe** [comment]
Looks good for QA credits

**Comment by terracoda** [comment]
@jessegreenberg, re-opening this issue as we need to add **Brianna Tomlinson** to the end team list as well. Apologies we missed this before you addressed this issue. 

**Current Screen Shot from today:**
<img width="620" alt="Screen Shot 2019-06-18 at 11 56 41 AM" src="https://user-images.githubusercontent.com/1902116/59672959-c5cff780-91c0-11e9-81ff-5ead706adeb6.png">

**Comment by terracoda** [comment]
@jessegreenberg, I just added a top priority so the issue is not missed ;-)

**Comment by jessegreenberg** [comment]
Not a problem at all, Brianna has been added to credits in master and release branches.
